### PR TITLE
Fix for WFLY-13967, Provide an simple overview of what a layer is at the start of the layers table

### PR DESCRIPTION
**Pull request by jfdenise**
Issue: https://issues.redhat.com/browse/WFLY-13967
@bstansberry , when this is good, I will open PR against 21.x.
JF

### CODE REVIEW

**Review by xstefank — APPROVED** [review]
+1

File: docs/src/main/asciidoc/_galleon/Galleon_layers.adoc [hunk 1]
```diff
@@ -1,6 +1,15 @@
 [[wildfly_layers]]
 === WildFly Layers
 
+A Galleon layer is a name that identifies a server capability (e.g.: jaxrs, 
+ejb, microprofile-config, jpa) or an aggregation of such capabilities. A layer captures a server capability in the form of:
+
+* A piece of server XML configuration (e.g.: extension, configured subsystem, interfaces) that describes the capability.
+* A set of JBoss modules that implements the capability.
```
Comment (bstansberry) [inline docs/src/main/asciidoc/_galleon/Galleon_layers.adoc:8]: s/JBoss modules/JBoss Modules modules/g

or just "modules".

More words, but perhaps

* A set of modules and other filesystem content that implements the capability.

Maybe that's too much information. Layers bring things other than modules is the point; e.g. core-tools, vault-tools etc.

Comment (jfdenise) [reply docs/src/main/asciidoc/_galleon/Galleon_layers.adoc]: @bstansberry , I focused on the modules/configs. I added what you suggested. I think it is vague enough to cover the "other files".

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
@jfdenise Thanks. Re a 21.x PR for that to be useful someone would have to rebuild the docs and update the wildfly.github.io site. We can do that if you'd like.

**Comment by jfdenise** [comment]
I was mentioning 21 because of the Fix Release of this fix. I am not sure it is important enough to release the web site.

**Pull request merged**